.coveragerc

```diff
@@ -23,6 +23,11 @@ omit =
     homeassistant/components/adguard/sensor.py
     homeassistant/components/adguard/switch.py
     homeassistant/components/ads/*
+    homeassistant/components/aemet/abstract_aemet_sensor.py
+    homeassistant/components/aemet/config_flow.py
+    homeassistant/components/aemet/sensor.py
```

Comment:
Do you need to add senor.py here. You do seem to have tests for it.

Reply:
If I remember correctly there were code coverage errors when I didn't add it.